frontend/src/tests/components/modalContent/__snapshots__/AddVarianceModal.spec.js.snap

```diff
@@ -147,23 +117,7 @@ ShallowWrapper {
               },
             ]
           }
-          coreUsers={
-            Array [
-              Object {
-                "core_user_guid": "51b3a499-a474-4d52-be99-5c5123d7501c",
-                "email": "user@gov.bc.ca",
-                "idir_user_detail": Object {
-                  "bcgov_guid": "13b7821b-04d9-4a1d-a0f4-8f6bba2c6094",
-                  "city": "Victoria",
-                  "department": "Mines Health, Safety & Enforcement Division",
-                  "title": "Geomatics Specialist",
-                  "username": "IDIR\\\\BLAH",
-                },
-                "last_logon": null,
-                "phone_no": "555-555-5556",
-              },
-            ]
-          }
+          inspectors={undefined}
```

Comment:
this smells? change other test to use your new mock?

Reply:
Set to mock data.